cvat/apps/dataset_manager/formats/pascal_voc.py

```diff
@@ -57,7 +57,8 @@ def _import(src_file, instance_data, load_data_callback=None):
                 shutil.move(f, anno_dir)
 
         dataset = Dataset.import_from(tmp_dir, 'voc', env=dm_env)
-        dataset.transform('masks_to_polygons')
+        if kwargs.get('conv_mask_to_poly', True):
```

Comment:
It doesn't look good to me to have a copy-pasted undocumented code fragment in almost all formats. Could you add some common class of function for this?

Reply:
@zhiltsov-max 
Could you please provide an example how the code should look like? 
It is not clear what you expect.

Reply:
It could look like a class:

```python
class MasksToPolygonsInImport:
    """
    Manages common logic for mask to polygons conversion in dataset import.
    This usecase is supposed for backward compatibility for the transition period.
    """

    @classmethod
    def declare_arg_names(cls):
        return ['conv_mask_to_poly']

    @classmethod
    def convert_dataset(cls, dataset, **kwargs) -> Dataset:
        if kwargs.get('conv_mask_to_poly', True):
            dataset.transform('masks_to_polygons')
        return dataset
```

Reply:
In this case we just incapsulate ``if`` logic.
Anyway need to modify each converter by calling ``convert_dataset``, right?

Reply:
Yes, exactly.